h/static/scripts/account/test/auth-controller-test.coffee

```diff
@@ -18,15 +18,36 @@ class MockSession
     $promise:
       finally: sandbox.stub()
 
-mockFlash = info: sandbox.spy()
-mockFormRespond = sandbox.spy()
-
-describe 'h:AuthController', ->
+# Return an AuthController instance and associated objects for tests to use.
+_controller = (mocksession) ->
+  controller = null
   $scope = null
   $timeout = null
-  auth = null
   session = null
-  $controller = null
+  mockFlash = info: sandbox.spy()
+  mockFormRespond = sandbox.spy()
+  inject(($controller, $rootScope) ->
+    session = mocksession or new MockSession()
+    $scope = $rootScope.$new()
```

Comment:
Is there a reason why you couldn't go the whole hog and make `$scope` and `$timeout` simply locals as well? Why are we continuing to close over these?